app/main-process/projectWindow.js

```diff
@@ -204,13 +204,19 @@ ProjectWindow.setViewSettingsChanged = function(f) {
 
 ProjectWindow.getViewSettings = function() {
     if(!fs.existsSync(veiwSettingsPath)) {
+        fs.writeFileSync(veiwSettingsPath, JSON.stringify({ theme:'light', zoom:'100' }), {
+            encoding: "utf-8"
+        });
```

Comment:
If the file doesn't exist [yet], I don't think we need to create it until the setting is actually changed so I don't see the need for this.